diff --git a/Makefile.base.mk b/Makefile.base.mk
index 90c22da..a615503 100644
--- a/Makefile.base.mk
+++ b/Makefile.base.mk
@@ -90,7 +90,7 @@ endif
 # Set build and link flags
 
 BASE_FLAGS = -Wall -Wextra -pipe -MD -MP
-BASE_OPTS  = -O3 -ffast-math -mtune=generic -msse -msse2 -fdata-sections -ffunction-sections
+BASE_OPTS  = -O3 -ffast-math -march=native -mcpu=native -fdata-sections -ffunction-sections
 
 ifeq ($(MACOS),true)
 # MacOS linker flags
